packages/examples/src/shape-distance/rect-line-dist.style

```diff
@@ -18,7 +18,7 @@ forall Text `g` {
   }
 }
 
-forall Line l {
```

Comment:
No this was not just for testing - the whole block originally read,

```
forall Line l {
  l.shape = Line {
    strokeColor: #000
  }

  ensure touching(`g`.shape, l.shape)
}
```

The reference to `g` was broken since it wasn't declared in the header.

Reply:
somehow missed this in the diffs. makes total sense!